packages/expo-age-range/android/src/main/java/expo/modules/agerange/AgeRangeExceptions.kt

```diff
@@ -0,0 +1,8 @@
+package expo.modules.agerange
+
+import expo.modules.kotlin.exception.CodedException
+
+internal class AgeRangeTaskCancelledException : CodedException("Age range task cancelled.")
+
+internal class AgeRangeDeniedException :
+  CodedException("ERR_AGE_RANGE_USER_DECLINED", "Age signals request was denied by parental controls.", null)
```

Comment:
We don't use these error code anymore. This is fine 
```suggestion
  CodedException("Age signals request was denied by parental controls.")
```

Reply:
turns out this is a leftover, unused class. Removing it altogether.